ffs/integrationtest/integration_test.go

```diff
@@ -774,7 +774,7 @@ func TestRenew(t *testing.T) {
 	ra := rand.New(rand.NewSource(22))
 	cid, _ := addRandomFile(t, ra, ipfsAPI)
 
-	renewThreshold := 50
+	renewThreshold := 12600
```

Comment:
Magic number!, considering that Lotus does multiple calculations and buffering to calculate when the deals will end. This number isn't that magic, I could explain it but will be quite long and a bit pointless for Powergate scope.

Reply:
Magic numbers = more value for powergate :)